server/integration-test/src/test/java/com/oceanbase/odc/service/flow/FlowResponseMapperFactoryTest.java

```diff
@@ -148,6 +148,14 @@ public void generateFlowInstanceDetailResp_entityInput_returnDesp() {
         Assert.assertEquals(instanceEntity.getId(), detailResp.getId());
     }
 
+    private TaskEntity getValidRandomTaskEntity() {
+        TaskEntity taskEntity = TestRandom.nextObject(TaskEntity.class);
```

Comment:
why not set to a specifica taskType,
we should avoid involve random object if the random value will affect test result

Reply:
Here is a bug left by history: If the task type is PRE_CHECK, then exception will be thrown in FlowResponseMapperFactory. You are right. Now I fix it by set the task type to ASYNC